project_generator/templates/makefile_gcc.tmpl

```diff
@@ -175,6 +175,7 @@ clean:
 	$(RM) $(TARGET).$(TARGET_EXT) $(TARGET).bin $(TARGET).map $(OBJ_FOLDER)*.* $(OBJ_FOLDER)
 	@echo ' '
 
-
+{% block extra_targets %}
```

Comment:
I assume what is it, because we are missing to provide extra options, that an app can supply :)